src/azure-cli/azure/cli/command_modules/backup/custom.py

```diff
@@ -541,11 +548,42 @@ def list_recovery_points(cmd, client, resource_group_name, vault_name, item, sta
     if use_secondary_region:
```

Comment:
To check whether while listing RPs from secondary region we should allow tier/archive related filters or not.